autogen/bundlegroups/sc.fiji.xml

```diff
@@ -0,0 +1,21 @@
+<!-- trove4j bundle -->
+<bundlegroup name="io.scif">
+
+	<!-- SCIFIO -->
+	<bundle name="fiji-bigdataviewer" version="${bigdataviewer-core.version}">
+		<artifacts>
+			<artifact>
+				<groupId>sc.fiji</groupId>
+				<artifactId>bigdataviewer-core</artifactId>
+				<version>${bigdataviewer-core.version}</version>
+			</artifact>
+		</artifacts>
+		<dependencies>
+			<bundleref name="net.imglib2:imglib2" version="${imglib2.version}" />
```

Comment:
trove4j is not declared as an dependency, as far as I can see in the pom of bigdataviewer: 
https://github.com/bigdataviewer/bigdataviewer-core/blob/master/pom.xml